content/docs/details/techdocs/index.md

```diff
@@ -0,0 +1,248 @@
+
+![white graph background](./white-graph.png)
+
+which could be defined in TechDocs with following code:
+
+```
+{% dot attack_plan.svg
+    digraph G {
+        rankdir=LR
+        Earth [peripheries=2]
+        Mars
+        Earth -> Mars
+    }
+%}
+```
+
+Adding 'bgcolor' attribute in the Graphviz code above (so its final form is):
+
+```
+{% dot attack_plan.svg
+    digraph G {
+        bgcolor="lightblue"
+        rankdir=LR
+        Earth [peripheries=2]
+        Mars
+        Earth -> Mars
+    }
+%}
+```
+
+will result in graph being rendered in lightblue color.
+
+
+![lightblue graph in techdocs](./lightblue-graph.png)
+
+This way you can customize the graph adding or removing any attribute you want.
+
+
+## Using a Documentation Monorepo
+
+If you need to have a place to store docs that are not related to a specific codebase or component, you may want to use a single repository to collect that meta documentation.
+
+Nested file structures and sub-directories can be modeled using the [Monorepo plugin for techdocs](https://github.com/backstage/mkdocs-monorepo-plugin).
+
+1. Add to your root `mkdocs.yaml` file.
+```yaml
+plugins:
+  - monorepo
+```
+
+2. Reference other `mkdocs.yaml` files in sub-directories using the `!include` syntax like so:
+
+```yaml
+nav:
+  - Intro: 'index.md'
+  - Authentication: 'authentication.md'
+  - API:
+    - v1: '!include ./v1/mkdocs.yml'
+    - v2: '!include ./v2/mkdocs.yml'
+```
+
+Standalone repos for documentation not related to a codebase or component should still be modeled in Roadie Backstage
+with a catalog-info.yaml file. The docs repo entity can be described with the following kind and spec type:
+
+```yaml
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+...
+spec: 
+  type: documentation
+...
+```
+
+## Including existing markdown files
+
+Mkdocs only processes markdown files in the mkdocs directory (defaults to `docs/`). If you've existing docs
+which live closer to the code e.g. a README.md in the root or at some other level they can't be referenced in the mkdocs.yaml.
+
+It is possible to use the [PyMdown snippets extension](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippets) to include markdown files from outside the mkdocs docs directory in markdown files within it.
+
+To enable this extension update your mkdocs.yaml:
+
+```yaml
+markdown_extensions:
+  # ... other extensions 
+  - pymdownx.snippets:
+      check_paths: true
+```
+
+Then you can include snippets in your markdown files. If a snippet is the only content in a file then the content is replaced
+by the referenced file. For example, to include a file TEST.md at the root of our repo we could do the following:
+- Create a file under the docs directory e.g. `docs/test.md`
+- Add a snippet to test.md (the snippet path is relative to the mkdocs.yaml file)
```

Comment:
The sentence inside parenthesis is useful as a statement in itself. Probably worth moving it to its own line or N.B